pandas/tests/arrays/interval/test_ops.py

```diff
@@ -4,8 +4,12 @@
 
 from pandas import Interval, IntervalIndex, Timedelta, Timestamp
 from pandas.core.arrays import IntervalArray
+from pandas.tests.scalar.interval.test_ops import overlaps_cases
 import pandas.util.testing as tm
 
+# silence "imported but unused" warning by flake8
```

Comment:
instead of this move the cases to a conftest

Reply:
done